runtime/src/main/java/org/corfudb/util/serializer/DynamicProtobufSerializer.java

```diff
@@ -103,9 +103,15 @@
     public DynamicProtobufSerializer(CorfuRuntime corfuRuntime) {
         this.type = ProtobufSerializer.PROTOBUF_SERIALIZER_CODE;
 
-        // Create and register a protobuf serializer to read the table registry.
-        ISerializer protobufSerializer = createProtobufSerializer();
-        Serializers.registerSerializer(protobufSerializer);
+        // Create or get a protobuf serializer to read the table registry.
+        ISerializer protobufSerializer;
+        try {
+            protobufSerializer = Serializers.getSerializer(ProtobufSerializer.PROTOBUF_SERIALIZER_CODE);
```

Comment:
Can we please fold this error message in Serializers.java into the exception itself so it does not get logged?
log.error("Serializer with type code {} not found. Please check custom serializers " +
                "for this client.", type.intValue());
as part of this change?

Reply:
Serializers.getSerializer() has already included the error message inside the SerializerException. Removed log.error() in Serializers.getSerializer().